### ISSUE TITLE
Does this package support Python 3?

### ISSUE THREAD

**Issue by cclauss** [wndhydrnt/python-oauth2#58]
On 19 February 2017, Python 3 will be 3,000 days old!  We are interested to see if we can get **at least 50%** of the [Top 5,000 PyPI packages](http://python3wos.mybluemix.net/5000) to compatible with Python 3 by that date.  We are *really* close and given that this package is in the PyPI Top 5,000, we seek your assistance in pushing us over that threshold.

  So, if this package *already supports Python 3* then please consider adding an appropriate [trove classifier](https://pypi.python.org/pypi?%3Aaction=list_classifiers) "*Programming Language :: Python :: 3*" to this package's PyPI page so that tools can automaticly determine its Python 3 compatibility.

**Comment by Trii** [comment]
@cclauss `setup.py` lists python 3 https://github.com/wndhydrnt/python-oauth2/blob/master/setup.py#L31-L35

**Comment by cclauss** [comment]
You are correct and https://pypi.python.org/pypi/oauth2 says the same.  Thanks.

**Comment by Trii** [comment]
That is strange @cclauss! @wndhydrnt did you see this?
> You are correct but https://pypi.python.org/pypi/python-oauth2 does not say the same. Thanks.

**Comment by wndhydrnt** [comment]
@Trii The package @cclauss was referring to was a different package. It's true that the version information is missing on PyPI. I always thought it would get set if its specified in `setup.py` but never actually checked it. i'm reopening this issue and check how to specify versions on PyPI.
